docs/user-guide/api-mediation/api-mediation-multi-tenancy.md

```diff
@@ -0,0 +1,278 @@
+# Zowe MultiTenancy Configuration
+
+## Component Layout example
+
+The following bullets describe which Zowe components should be enabled and disabled in the multi-tenancy environment.
```

Comment:
I think it would be useful to include information about why certain components must be enabled or disabled.